docs/docsource/operation/Orbit_UI/Dashboard.rst

```diff
@@ -8,9 +8,26 @@ On login, the dashboard displays:
 The dashboard provides useful information about ongoing Zenko
 operations, replication status, and cloud server health.
 
-In addition to the options on the left navigation pane, you can click
-the **View More Statistics** button to see more statistics about
-buckets, objects, and resources.
+The Sidebar
+-----------
+
+Visible from all Orbit windows, the sidebar contains navigational
+buttons for:
+
+   * :ref:`Statistics`
+   * :ref:`Location Status<location_status>`
+   * :ref:`Storage Accounts <add_a_new_user>`
+   * :ref:`Storage Locations <orbit_add_location>`
+   * :ref:`Endpoints <endpoint>`
+   * :ref:`Browser <create_a_bucket>`
+   * :ref:`Search<orbit_md_search>`
```

Comment:
minor: for consistency, you might add a spacing like `Search <orbit_md_search>` here and below.

Reply:
done.